source/agora/consensus/Ledger.d

```diff
@@ -243,6 +243,28 @@ public class Ledger
         return result;
     }
 
+    /***************************************************************************
+
+        Get the count of active validators at a given height.
+
+        This function is more efficient than checking the length of the array
+        returned by `getValidators` as it avoids any intermediate allocation.
+        An 'active' validator is one that hasn't been slashed,
+        and may sign a block. A validator that enrolls, reveal all its
+        pre-images but never sign a block will still be counted as 'active'.
+
+        Params:
+          height = Height at which to query the validator set.
+                   Accurate results are only guaranteed for
+                   `height <= this.getBlockHeight() + 1`.
+
```

Comment:
Should have a doc for a return value

Reply:
Fixed in https://github.com/bosagora/agora/pull/2805